python/lsst/ts/observatory/control/data/mtcalsys.yaml

```diff
@@ -87,3 +87,20 @@ laser_functional:
   use_fiberspectrograph_blue: false
   exposure_times:
     - 15.0
+
+cbp_calibration:
+  calib_type: CBPCalibration
+  use_camera: false
+  wavelength: 700
+  wavelength_width: 800
+  wavelength_resolution: 50
+  use_cbpelectrometer: true 
```

Comment:
I suppose we should rename the other electrometer to be flatfieldelectrometer or something like that.

Reply:
Also, `use_electrometer` is required currently. You'll have to add these two new ones to mtcalsys_schema.yaml